src/main/java/data_access/DBUserDataAccessObject.java

```diff
@@ -35,10 +32,11 @@
  */
 public class DBUserDataAccessObject implements SignupUserDataAccessInterface,
         LoginUserDataAccessInterface,
-        HomeUserDataAccessInterface, My_ReviewsDataAccessInterface,
-        LogoutUserDataAccessInterface, WatchlistsUserDataAccessInterface,
-        WatchlistUserDataAccessInterface, RecommendationsUserDataAccessInterface, Survey1UserDataAccessInterface {
```

Comment:
The WatchlistUserDataAccessInterface should not have been removed, this is what caused the issue in the WatchlistInteractor